src/app/components/accountRow/index.tsx

```diff
@@ -315,6 +329,21 @@ function AccountRow({
                   <CaretDown weight="bold" size={16} />
                 )}
               </CurrentAccountTextContainer>
+              {isAccountListView && totalBalance && (
+                <NumericFormat
+                  value={totalBalance}
+                  displayType="text"
+                  prefix={`${currencySymbolMap[fiatCurrency]}`}
+                  thousandSeparator
+                  renderText={(value: string) => <Balance isSelected={isSelected}>{value}</Balance>}
+                />
+              )}
+              {isAccountListView && !totalBalance && (
+                <Balance isSelected={isSelected}>
+                  {EMPTY_LABEL}
+                  <MoonLoader color="white" size={12} />
+                </Balance>
+              )}
```

Comment:
question: any reason this conditions aren't ternary?

Reply:
I think it would require to make a nested ternary condition here which is usually considered not a good practice by code linters